Selection_Sort_C/main.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+
+void Swap(int *left, int *right)
+{
+    int temp = *left;
+    *left = *right;
+    *right = temp;
+}
+
+void SortArray(int array[], int sizeOfArray)
+{
+    int minElement = 0;
+    int thisElement = 0;
+
+    for(int i = 0; i < sizeOfArray - 1; i++)
+    {
+        minElement = i;
+
+        for(thisElement = i; thisElement < sizeOfArray; thisElement++)
+        {
+            if(array[thisElement] < array[minElement])
+            {
+                Swap(&array[minElement], &array[thisElement]);
+            }
+        }
+    }
+
+    printf("Sorted array:\n");
```

Comment:
Please create a separate function(In C we call them functions) for printing the array.
Please notice, that your method is called SortArray and not SortAndPrintArray, so it shouldn't print anything.
When you implement method PrintArray, please call it from the Main method, after you sorted the array.

Reply:
Done.